masterlist.yaml

```diff
@@ -17642,6 +17642,11 @@ plugins:
         name: 'Carriage and Ferry Travel Overhaul'
     after: [ 'Immersive Citizens - AI Overhaul.esp' ]
     msg:
+      - <<  *alsoUseX
```

Comment:
There must be a `:` after the `<<` followed by one empty space (currently there are two).